main/src/main/java/com/sedmelluq/discord/lavaplayer/source/AudioSourceManagers.java

```diff
@@ -40,6 +41,7 @@ public static void registerRemoteSources(AudioPlayerManager playerManager, Media
         playerManager.registerSourceManager(new TwitchStreamAudioSourceManager());
         playerManager.registerSourceManager(new BeamAudioSourceManager());
         playerManager.registerSourceManager(new GetyarnAudioSourceManager());
+        playerManager.registerSourceManager(new NicoAudioSourceManager());
```

Comment:
this might catch people off guard, not sure how I feel about this

Reply:
Equally, `registerRemoteSources` is expected to enable all sources Lavaplayer has to offer. The only reason this wasn't the case for Nico in the past was because it required credentials which couldn't be defaulted. I think the majority of users won't care what this registers as long as it doesn't introduce security vulnerabilities or similar. Those who want fine-grained control over what the library does always has the option of manually registering each source manager.

Reply:
then we should at least mention it in the release notes specifically